examples/convert_group_studies.py

```diff
@@ -63,13 +65,15 @@
 for subject_id in subject_ids:
     subject = 'sub%03d' % subject_id
     for run in runs:
-        raw_file = op.join(data_path, repo, subject, 'MEG',
-                           'run_%02d_raw.fif' % run)
-
-        # Make it BIDS compatible
-        raw_to_bids(subject_id='%02d' % subject_id, session_id='01', run=run,
-                    task='VisualFaces', raw_file=raw_file, event_id=event_id,
-                    output_path=output_path, overwrite=True)
+        raw_fname = op.join(data_path, repo, subject, 'MEG',
+                            'run_%02d_raw.fif' % run)
+        raw = mne.io.read_raw_fif(raw_fname)
```

Comment:
I think this should use `read_raw` for the reading of the raw file.
Because KIT data *has* to use the `read_raw` function (a point that should probably be mentioned somewhere) I think it would be good to have the standard that we use `read_raw` everywhere to avoid any issues. (see other comments re this point...)